docs/user-guide/command-line-interface.md

```diff
@@ -69,7 +69,9 @@ Miscellaneous:
   --debug                    Output debugging information
   -h, --help                 Show help
   -v, --version              Outputs the version number
-  ```
+  --no-inline-config         Prevent comments from changing eslint rules -
```

Comment:
`--no-inline-config` is correct if by default its `true` but your option description says `false`. Thats why I am trying to make sure what it is?